File: create.rb
#!/usr/bin/env ruby


TEMPLATE ="""#!/usr/bin/env ruby

def main

end

main()

"""

def new
  n = ARGV.last.to_s
  FileUtils.mkdir n
  File.open("#{n}/#{n}.rb", 'w') do |f|
    f.puts TEMPLATE
  end
end

File: create.py
#!/usr/bin/env python

import sys, os

TEMPLATE = """#!/usr/bin/env python

def main():


if __name__ == '__main__':
  print main()

"""

def main():
  n = str(sys.argv[1])
  os.makedirs(n)
  f = open('{0}/{0}.py'.format(n), 'w')
  f.write(TEMPLATE)
  f.close()


if __name__ == '__main__':
  main()
